GiniVision/Classes/Core/Screens/Screen API Coordinator/GiniScreenAPICoordinator.swift

```diff
@@ -196,6 +199,16 @@ extension GiniScreenAPICoordinator {
 extension GiniScreenAPICoordinator {
     
     @objc func back() {
+        
+        switch screenAPINavigationController.topViewController {
+        case is CameraViewController:
+            trackingDelegate?.onCameraScreenEvent(event: Event(type: .exit))
+        case is AnalysisViewController:
+            trackingDelegate?.onAnalysisScreenEvent(event: Event(type: .cancel))
```

Comment:
Documentation should mention, that `.cancel` is triggered only when using the Screen API.